OpenEdXMobile/res/layout/row_course_card.xml

```diff
@@ -12,65 +12,61 @@
         android:layout_height="match_parent"
         android:contentDescription="@null"
         android:scaleType="fitXY"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         tools:src="@drawable/placeholder_course_card_image" />
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_gravity="bottom"
         android:background="@color/transparent_white_85"
-        android:minHeight="@dimen/course_detail_card_height"
-        android:orientation="horizontal">
+        android:orientation="vertical"
+        android:paddingLeft="@dimen/edx_default_margin"
+        android:paddingTop="@dimen/edx_half_margin"
+        android:paddingRight="@dimen/edx_default_margin"
+        android:paddingBottom="@dimen/edx_default_margin"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/header_image_view">
 
-        <LinearLayout
-            android:layout_width="0dp"
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/course_detail_name"
+            style="@style/course_outline_card_title"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_gravity="bottom"
-            android:layout_weight="1"
-            android:orientation="vertical"
-            android:padding="@dimen/widget_margin">
+            tools:text="Programming for Data Science" />
 
-            <TextView
-                android:id="@+id/course_detail_name"
-                style="@style/course_card_title"
-                tools:text="Course Name" />
-
-            <TextView
-                android:id="@+id/course_detail_extras"
-                style="@style/course_card_course_run"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                tools:text="XX | xx | xxxxx" />
-        </LinearLayout>
+        <com.google.android.material.textview.MaterialTextView
```

Comment:
why do we need to replace it with `MaterialTextView`.

Reply:
There was no need. I just updated the view.